sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/FilterScanConverter.java

```diff
@@ -47,7 +46,7 @@ public RelNode convert(ResolvedFilterScan zetaNode, List<RelNode> inputs) {
                 zetaNode.getFilterExpr(),
                 zetaNode.getInputScan().getColumnList(),
                 input.getRowType().getFieldList(),
-                ImmutableMap.of());
+                context.getFunctionArgumentRefMapping());
```

Comment:
Should we should change all converters to use `getFunctionArgumentRefMapping`?

Reply:
I am planning to improve testing coverage in next steps and gradually make necessary converters to use `getFunctionArgumentRefMapping`. E.g. in project or join.